jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -533,7 +536,26 @@ public void onResize() {
     }
 
     @Override
-    public ColumnSortList reloadColumnSortList() {
+    public String getSortColumn() {
+        final ColumnSortList columnSortList = getListGrid().getColumnSortList();
+        if (columnSortList != null && columnSortList.size() == 1) {
+            return columnSortList.get(0).getColumn().getDataStoreName();
+        } else {
+            return null;
+        }
+    }
+
+    @Override
+    public Boolean isSortAscending() {
+        final ColumnSortList columnSortList = getListGrid().getColumnSortList();
+        if (columnSortList != null && columnSortList.size() == 1) {
```

Comment:
if we add limit size is 1, we do not need check columnSortList.size() == 1

Reply:
would rather keep on the safe side just in case we might get a list with size 0, that would throw an exception.

Reply:
ok